app-dicts/mecab-ipadic/mecab-ipadic-2.7.0.20070801.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
```

Comment:
Overall, please revbump for new EAPI ports, because it's an opportunity to flush old ones out of the VDB (/var/db/pkg) and one day we'd like to drop support for old EAPIs in Portage (at least < EAPI 7, as EAPI 7 made a lot of things better and simpler).

Reply:
Gotcha, I'll go back and see if repology says there's any bumps and look on the project's site. I did notice one revbump for `app-dicts/mecab-unidic` that used a different SRC_URI but I wasn't sure if that was the same one as the one currently being used. [Repology](https://repology.org/project/mecab-unidic/versions) of the package if you want to look at it.